types/lesgo/utils/validateFields.d.ts

```diff
@@ -1,12 +1,12 @@
 export interface Validation<T> {
-    key: T;
-    type: "string" | "object" | "number" | "decimal" | "email" | "array" | "enum";
-    required?: boolean;
+	key: T;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/lesgo/utils/validateFields.d.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.